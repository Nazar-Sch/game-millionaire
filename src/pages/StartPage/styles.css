.start-container {
  background: linear-gradient(to bottom right, #ffffff 50%, var(--orange-5-color) 50%);
}
