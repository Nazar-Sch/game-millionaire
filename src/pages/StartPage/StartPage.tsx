import { FC } from 'react';
import StyledLink from '../../components/StyledLink';
import { cx } from '../../utils';
import './styles.css';

const StartPage: FC = () => {
  return (
    <div className={cx('custom-page-container', 'start-container')}>
      <img src='/finger.png' alt="Millionaire's finger" className='custom-image' />
      <h2 className='custom-heading'>Who wants to be a milliomaire?</h2>
      <StyledLink className='custom-button' to='/game'>
        Start
      </StyledLink>
    </div>
  );
};

export default StartPage;
