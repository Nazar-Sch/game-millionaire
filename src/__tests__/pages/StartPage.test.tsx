import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import StartPage from '../../pages/StartPage/StartPage';

describe('StartPage', () => {
  it('should render correctly', () => {
    const { container } = render(
      <MemoryRouter>
        <StartPage />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
