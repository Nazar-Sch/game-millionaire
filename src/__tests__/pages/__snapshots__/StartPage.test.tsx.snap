// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartPage should render correctly 1`] = `
<div>
  <div
    class="custom-page-container start-container"
  >
    <img
      alt="Millionaire's finger"
      class="custom-image"
      src="/finger.png"
    />
    <h2
      class="custom-heading"
    >
      Who wants to be a milliomaire?
    </h2>
    <a
      class="shared-link custom-button"
      href="/game"
    >
      Start
    </a>
  </div>
</div>
`;
